src/lib/token.js

```diff
@@ -13,8 +13,12 @@ export function hasLocalToken() {
 }
 
 // Sets token in local storage
-export function setLocalToken() {
-  const token = getToken();
+export function setLocalToken(token) {
+  token = token || getToken();
   const myStorage = window.localStorage;
   myStorage.setItem('token', token);
```

Comment:
I believe we can simplify this to a one liner if by doing

```js
window.localStorage.setItem('token', token || getToken());
```

I don't see much value of moving `token` and `window.localStorage` into their own variables  just to use them in the next line. 😄

Reply:
Hi Nabil and Megan, I ran the project on my localhost, checked through everything and it works the way it's stated in the AC, good job!